vips/vip-498/bscmainnet-2.ts

```diff
@@ -0,0 +1,254 @@
+      },
+      {
+        target: ethereum.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [USDe, [ethereum.REDSTONE_ORACLE, ethereum.CHAINLINK_ORACLE, ethereum.CHAINLINK_ORACLE], [true, true, true]],
+        ],
+        dstChainId: LzChainId.ethereum,
+      },
+
+      // Market configurations
+      {
+        target: VsUSDe_CORE,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [sUSDe, parseUnits("10000", 18), ethereum.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [ethereum.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [ethereum.POOL_REGISTRY, sUSDe_INITIAL_SUPPLY],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VsUSDe_CORE,
+            parseUnits("0.72", 18), // CF
+            parseUnits("0.75", 18), // LT
+            sUSDe_INITIAL_SUPPLY, // initial supply
+            ethereum.NORMAL_TIMELOCK,
+            parseUnits("20000000", 18), // supply cap
+            parseUnits("0", 18), // borrow cap
+          ],
+        ],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: VsUSDe_CORE,
+        signature: "transfer(address,uint256)",
+        params: [ethers.constants.AddressZero, parseUnits("86", 8)], // around $100
+        dstChainId: LzChainId.ethereum,
+      },
+      (() => {
+        const vTokensMinted = convertAmountToVTokens(USDe_INITIAL_SUPPLY, parseUnits("1", 28));
+        const vTokensRemaining = vTokensMinted.sub(parseUnits("86", 8));
+        return {
+          target: VsUSDe_CORE,
+          signature: "transfer(address,uint256)",
+          params: [VTOKEN_RECEIVER, vTokensRemaining],
+          dstChainId: LzChainId.ethereum,
+        };
+      })(),
+      {
+        target: VsUSDe_CORE,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.05", 18)],
+        dstChainId: LzChainId.ethereum,
+      },
+      // Add USDe Market
+      // oracle config
+      {
+        target: ethereum.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            sUSDe,
+            [sUSDe_ERC4626ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+          ],
+        ],
+        dstChainId: LzChainId.ethereum,
+      },
```

Comment:
I think this is not needed, because the oracle for `sUSDe` is already configured on mainnet

Reply:
that's right . 45d04faa11d6bfef72345e21f222d0da1074c7fd